### PR TITLE
Linear regression

### PR DESCRIPTION
**Pull request by thunterdb**
Adds a specific check for linear regression with the L-BFGS solver.

Tested on a small amount of data.

### CODE REVIEW

File: src/main/scala/configs/mllib-small.yaml [hunk 1]
```diff
@@ -63,3 +63,9 @@ benchmarks:
       numClasses: 4
       numFeatures: 5
       maxIter: 3
+  - name: regression.LinearRegression
+    params:
+      numFeatures: 100
```
Comment (jkbradley) [inline src/main/scala/configs/mllib-small.yaml]: numExamples, numTestExamples?

**Comment by jkbradley** [comment]
Just the 1 comment

File: src/main/scala/configs/mllib-small.yaml (hunk 1 above)
Comment (thunterdb) [inline src/main/scala/configs/mllib-small.yaml]: Added

**Comment by thunterdb** [comment]
Thanks, merging

**Comment by jkbradley** [comment]
LGTM, you can merge

**Pull request merged**